mmcls/models/backbones/levit.py

```diff
@@ -0,0 +1,586 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import itertools
+
+import torch
+import torch.nn as nn
+from mmcv.cnn import Linear, build_conv_layer, build_norm_layer
+from mmengine.model import BaseModule
+
+from mmcls.models.backbones.base_backbone import BaseBackbone
+from mmcls.models.heads.levit_head import LeViTClsHead
+from mmcls.registry import MODELS
+
+
+class hybrid_cnn(BaseModule):
```

Comment:
The name of classes should obey [Google python style](https://zh-google-styleguide.readthedocs.io/en/latest/google-python-styleguide/python_style_rules/#section-15)

Reply:
Modified